src/Compilers/Core/Portable/Syntax/AbstractWarningStateMap.cs

```diff
@@ -71,6 +74,7 @@ protected struct WarningStateMapEntry : IComparable<WarningStateMapEntry>
             public readonly int Position;
 
             // the general option applicable to all warnings, accumulated of all #pragma up to the current Line.
+            [MaybeNull]
             public readonly WarningState GeneralWarningOption;
```

Comment:
You don't need to change this right now, but this doesn't follow the [naming conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/generics/generic-type-parameters#type-parameter-naming-guidelines) for type parameters and makes it harder to tell at a glance why `[MaybeNull]` was needed.

Reply:
Yeah that really threw me off toe. Going to rename it.